recipe/meta.yaml

```diff
@@ -47,6 +56,7 @@ outputs:
         - "x86_64-conda_cos6-linux-gnu/sysroot/lib/libpthread-2.12.2.so"
     requirements:
       run:
+        - {{ pin_subpackage("sysroot_" ~ ctng_target_platform, exact=True) }}
```

Comment:
This should only be in `gcc_impl_*` output. Can you remove it from others?

Reply:
Did you mean to mark this as resolved @isuruf? I see this pin in the other outputs still.